include/uapi/sound/sof/ext_manifest.h

```diff
@@ -0,0 +1,61 @@
+/* SPDX-License-Identifier: ((GPL-2.0 WITH Linux-syscall-note) OR BSD-3-Clause) */
```

Comment:
typo in commit message : "built-time" -> build time?

Reply:
changed

Reply:
I still dont see the change?

Reply:
@ktrzcinx Yup, it's still "store built time" for me as well.

Reply:
True, I've lost that one. Changed.